src/ui/components/Widgets/Chart/hooks/useLoadingChart.ts

```diff
@@ -301,17 +301,35 @@ export const useLoadingChart = (props: LoadingChartHookProps) => {
                 dispatch({type: WIDGET_CHART_RESET_CHANGED_PARAMS, payload: newChangedParams});
             }
 
+            let widgetConfig = {};
+            if (enableActionParams) {
+                widgetConfig = {
+                    widgetConfig: {
```

Comment:
widgetConfig is somehow strangely duplicated here. Why is he inside?

Reply:
fixed